physionet-django/project/models.py

```diff
@@ -608,6 +611,88 @@ def get_author_info(self, separate_submitting=False, include_emails=False):
             else:
                 return authors
 
+    def get_platform_citation(self):
+        """
+        Returns the information needed to generate the standard platform
+        citation in multiple formats (MLA, APA, Chicago, Harvard, and
+        Vancouver).
+
+        1. MLA (8th edition) [https://owl.purdue.edu/owl/research_and_citation/
+                              mla_style/mla_formatting_and_style_guide/
+                              mla_formatting_and_style_guide.html]
+        2. APA (7th edition) [https://owl.purdue.edu/owl/research_and_citation/
+                              apa_style/apa_style_introduction.html]
+        3. Chicago (17th edition) [https://owl.purdue.edu/owl/
+                                   research_and_citation/
+                                   chicago_manual_17th_edition/
+                                   cmos_formatting_and_style_guide/
+                                   chicago_manual_of_style_17th_edition.html]
+        4. Harvard [https://www.mendeley.com/guides/harvard-citation-guide]
+        5. Vancouver [https://guides.lib.monash.edu/ld.php?content_id=14570618]
+
+        Parameters
+        ----------
+        N/A
+
+        Returns
+        -------
+        citation_styles [dict]:
+            dictionary containing the desired citation style
+        """
+        authors = [
+            'Ary Goldberger',
+            'Luis Amaral',
+            'Leon Glass',
+            'Jeffrey M Hausdorff',
+            'Plamen C Ivanov',
+            'Roger Mark',
+            'Joseph E Mietus',
+            'George B Moody',
+            'Chung Kang Peng',
+            'H Eugene Stanley'
+        ]
+
+        all_authors = []
+
+        for a in authors:
+            all_authors.append(PublishedAuthor(
+                user=User.objects.get(username='admin'),
```

Comment:
`username='admin'`?

Reply:
For some reason a user is required to create an instance of the PublishedAuthor class to use  its functions